api/src/controllers/put/inspection.js

```diff
@@ -0,0 +1,232 @@
+var mongoose = require('mongoose');
+var putUtils = require('../../utils/put-utils');
+
+// Example of incomingObj
+/**
+ *    inspections: [
+ *     {
+ *       _id: '85ce24e603984b02a0f8edb42a334876',
+ *       recordName: 'test abc',
+ *       recordType: 'whatever',
+ *       ...
+ *       InspectionLNG: {
+ *          description: 'lng description'
+ *          addRole: 'public',
+ *       }
+ *       InspectionNRCED: {
+ *          summary: 'nrced summary'
+ *          removeRole: 'public',
+ *       }
+ *     },
+ */
+exports.editMaster = async function (args, res, next, incomingObj) {
+    var _id = null;
+    if (!incomingObj._id) {
+        return {
+            status: 'failure',
+            object: incomingObj,
+            errorMessage: 'No _id provided'
+        }
+    } else {
```

Comment:
nitpick: dont need the else wrapper, because you return in the if anyways. 

Same for the cases further down.